src/main/java/dev/mlml/command/impl/Fox.java

```diff
@@ -0,0 +1,51 @@
+package dev.mlml.command.impl;
+
+import dev.mlml.Utils;
+import dev.mlml.command.Command;
+import dev.mlml.command.CommandInfo;
+import dev.mlml.command.Context;
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.utils.FileUpload;
+import net.dv8tion.jda.api.utils.data.DataObject;
+import net.dv8tion.jda.internal.utils.IOUtil;
+
+import java.io.InputStream;
+import java.net.URL;
+import java.util.Objects;
+
+@CommandInfo(
+        keywords = {"fox", "foxie"},
+        name = "Fox",
+        description = "Get a fox media",
+        permissions = {Permission.MESSAGE_SEND},
```

Comment:
The `permissions` field is this by default, I know it's in `Dog.java` but that is set to be removed and I'm gonna start enforcing it now.